mypy/test/testmodulefinder.py

```diff
@@ -143,14 +148,15 @@ def setUp(self) -> None:
             package_path,
             "modulefinder-site-packages",
         ))
+
+        egg_dirs, site_packages = expand_site_packages([self.package_dir])
+
         self.search_paths = SearchPaths(
             python_path=(),
             mypy_path=(
                 os.path.join(data_path, "pkg1"),
             ),
```

Comment:
Might as well clean up this line (It really doesn't need to be split across three lines).